Misc/NEWS.d/next/Core and Builtins/2023-08-11-16-44-29.gh-issue-107868.BgT7zE.rst

```diff
@@ -0,0 +1 @@
+Reduce `sum(range(...))` from `O(n)` to `O(1)` time . Patch by Marco Cognetta.
```

Comment:
```suggestion
Reduce ``sum(range(...))`` from ``O(n)`` to ``O(1)`` time . Patch by Marco Cognetta.
```
You should use double backticks in Rst.
Also, you can add yourself to `Misc/ACKS` file (please, follow the alphabetical order)